osf/models/node.py

```diff
@@ -124,7 +125,9 @@ def get_children(self, root, active=False):
                     root.pk])
                 row = cursor.fetchone()[0]
                 if not row:
-                    return AbstractNode.objects.none()
+                    return AbstractNode.objects.filter(id=root.pk) if include_root else AbstractNode.objects.none()
+                if include_root:
+                    row.append(root.pk)
```

Comment:
`include_root` argument added to get_children, so you can return an entire hierarchy plus the node at the top.